### DISCUSSION TITLE
How to remove listener ?

### DISCUSSION THREAD

**Discussion by ghost** [geckosio/geckos.io#190] [Help / Questions] [answered]
Hello, is there a way to remove listener that was added ?

On socket.io we can do it like that :

`socket.removeAllListeners("message");`

So can we do it on geckos.io ? If no, it would be cool if you can add it

I have too a question about how can I detect any message without taking care about the message name ? in socket.io it works like that : 

```js
socket.onAny(function() {
      //Do something
})
```

**Comment by yandeu** [comment, marked as answer]
I believe both are not possible. But both suggestions sound like a good idea.
